styless.css

```diff
@@ -0,0 +1,32 @@
+
+p {
+    color: 
```

Comment:
you have an error if you write a property you need to give it a value then end it with `;`